examples/marvel/analysis_options.yaml

```diff
@@ -0,0 +1,29 @@
+# This file configures the analyzer, which statically analyzes Dart code to
+# check for errors, warnings, and lints.
+#
+# The issues identified by the analyzer are surfaced in the UI of Dart-enabled
```

Comment:
This should be removed (same for todos).

The only thing this file does is disable lints, since Riverpod enables more lints than flutter_lints